lib/strategy.js

```diff
@@ -387,8 +385,12 @@ Strategy.prototype.configure = function(identifier, done) {
  * @return {Object}
  * @api protected
  */
-Strategy.prototype.authorizationParams = function(options) {
-  return {};
+AcpStrategy.prototype.authorizationParams = function(options) {
+	let params = {};
+	const prompt = options.prompt;
+	params = prompt ? Object.assign(params, {prompt}) : params;
```

Comment:
if you're sticking to es5, you must use `{ prompt: prompt }`. Shorthand property names are ES6.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer